server/claim-review-task/claim-review-task.controller.ts

```diff
@@ -33,7 +33,11 @@ export class ClaimReviewController {
         const { page = 0, pageSize = 10, order = 1, value } = getTasksDTO;
         return Promise.all([
             this.claimReviewTaskService.listAll(page, pageSize, order, value),
-            this.claimReviewTaskService.count({ "machine.value": value }),
+            this.claimReviewTaskService.count(
+                value === "published"
```

Comment:
This logic seems to be repeated more then once, let's move this to a library (`getQueryMatchForMachineValue`) and share the code everywhere.